package.json

```diff
@@ -7,19 +7,24 @@
     "preview": "nuxt preview",
     "format": "prettier --write .",
     "format-check": "prettier --check .",
-    "lint": "eslint ."
+    "lint": "eslint .",
+    "test": "vitest --config vite.config.test.ts --environment jsdom"
```

Comment:
For what it's worth, I think you can simplify this to `"test": "vitest run"`, and you can set some of the properties in the vitest config file, or in the vite config file like this:
https://github.com/bcgov/bcrs-shared-components/blob/eb235158fa6096d5ce13fc2eb4c387931d593d55/vite.config.ts#L27